internal/plugins/manage.go

```diff
@@ -68,8 +68,7 @@ func GetPluginYamls(uri, pluginName string) (PluginSpecs, error) {
 	githubContents = githubContents.filter(fileType)
 	var pluginSpecs PluginSpecs
 	for _, githubContent := range githubContents {
-		// Only look at the spec yaml files
-		if strings.HasSuffix(githubContent.Name, ".yaml") {
+		if strings.EqualFold(githubContent.Name, DefaultPluginFileName) {
```

Comment:
Not convinced if we need insensitivity to begin with. Might better to start out as strict case and then relax later if need arises.